### PR TITLE
[WIP] - (Hardlight) Bow Update - i'm going to be away for the next 2 days reopening when ready

### PR DESCRIPTION
**Pull request by silicons**
Totally small PR. Not finalized
:cl:
rscadd: Bows have been readded! They can (normally) nock a single arrow, but only if it is currently being held. To fire them, you have to hold down your mouse to draw it back, and release to fire. The arrow will only fire if the bow is sufficiently drawn, and will be scaled based on how far drawn back it is. Bows are two-handed required weapons.
experimental: Hardlight Energy Longbows have been re-added to nuclear operatives and traitors. They are able to fabricate [different types of] arrows directly onto their bows with special gloves with self-recharging internal energy capacitors. They can be retracted to fit in pockets when not in use, much like energy swords.
/:cl:
## stats are not finalized at all, i'm trying to make them the same power level as the ebow but not stun.
Tentative stats for syndicate versions - 12 TC.
Hardlight Energy Longbow - 2 seconds to max draw, 150 weight. 3 tile free directional zoom.
Gloves - 20 energy, recharges 2 per second.
number - number = minimum - maximum
Normal - 10 - 35 brute, 0.9 - 0.6 speed - 5 energy
Incendiary - 5 - 15 burn, 1 - 10 firestacks, 0.9 - 0.75 speed - 10 energy
Piercing - 10 - 25 brute, 0.7 - 0.45 speed, 2 pierce, 0.1 - 0.45 AP - 10 energy

### CODE REVIEW

**Comment by optimumtact** [comment]
are they still broken

**Comment by kevinz000** [comment]
@optimumtact this time they will NOT be broken and NOT pierce through half the station to the point where people get banned for using it.

**Comment by jakeramsay007** [comment]
But that's what made them fun, murdering half the station with endlessly ricocheting hardlight arrows!

**Comment by Kromgar** [comment]
We need DOUBLE longbows.

File: code/modules/projectiles/guns/misc/beam_rifle.dm [hunk 1]
```diff
@@ -183,6 +183,7 @@
 /obj/item/gun/energy/beam_rifle/Initialize()
 	. = ..()
 	current_tracers = list()
+	flags_2 |= SLOWS_WHILE_IN_HAND_2
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/projectiles/guns/misc/beam_rifle.dm]: ??

Comment (optimumtact) [reply code/modules/projectiles/guns/misc/beam_rifle.dm]: SLOWS WHILE IN HAND

**Comment by kevinz000** [comment]
@Spaceinaba and everyone else stats are still being determined.

File: code/modules/projectiles/guns/misc/beam_rifle.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/projectiles/guns/misc/beam_rifle.dm]: oh yeah beam rifles are supposed to slow the user when aiming but it didn't do that for some reason this fixes that ayyy

Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/guns/misc/beam_rifle.dm]: oh ok can you update CL?

**Comment by Bawhoppen** [comment]
Remove zoom

**Comment by Supermichael777** [comment]
What can I do with this that I cant do by raiding a O2 locker and getting atmos mad by making a pneumatic cannon? a deck of syndie cards or welder is legit competition for this stuffed into a much more deniable cannon.

**Comment by ExcessiveUseOfCobblestone** [comment]
are we going to arms race against an item that can theoretically oneshot rather than just remove the oneshot item?

**Comment by ExcessiveUseOfCobblestone** [comment]
necro...

**Pull request closed without merging**